canvas_modules/common-canvas/__tests__/common-canvas/common-canvas-test.js

```diff
@@ -175,7 +235,7 @@ describe("CommonCanvas renders correctly", () => {
 	});
 });
 
-function createCommonCanvas(config, canvasController, toolbarConfig, notificationConfig, handlers) {
+function createCommonCanvas(config, canvasController, canvasBottomPanelParams, canvasRightPanelParams, toolbarConfig, notificationConfig, handlers) {
```

Comment:
1. You don't need two objects to pass in the parameters. Just have one object that either contains a `showBottomPanel` field or a `showRightFlyout` field.
2. Put the parameter at the end of the set, after handlers
3. Make createCommonCanvas be able to work, even if the object being passed in is undefined.